package.json

```diff
@@ -196,6 +198,7 @@
         "react-intl": "^5.24.6",
         "react-leaflet": "^3.2.5",
         "react-markdown": "^8.0.0",
+        "react-monaco-editor": "^0.47.0",
```

Comment:
I prefer https://github.com/suren-atoyan/monaco-react 

You may want to suspect that a library that uses class syntax in the README's React code is an unmaintained library.

Reply:
Ok! I'll look into your suggested library.